lib/js/components/Divider/Divider.stories.mdx

```diff
@@ -0,0 +1,36 @@
+import { Meta, Story, Props } from '@storybook/addon-docs';
+import Divider from './Divider'; import { PROMINENCES } from './Divider.consts';
```

Comment:
Do You have prettier turned on? It should move import to the next line.

Reply:
I did not have it turned on. Fixed, thanks.